### ISSUE TITLE
Pull request: Add a comment selector to block yourself and share your block with the community.

### ISSUE THREAD

**Issue by gut1414** [MagicJinn/Block-Sponsor-Comments#29]
I suggest adding a built-in blocking tool to allow the user to block sponsored comments themselves. This is already the case with SponsorBlock for video sequences; involving the community makes it effective. 

I see a lot of unblocked sponsored comments and descriptions, because I'm not an English speaker and there are so many sponsors and languages, that we have to involve all volunteer users to detect and block what they consider sponsored. 

Note: I had to create a new issue as a suggestion, because I can't seem to create a suggestion. Github stops me and asks me to choose a version, compare with another of I don't know what... it's nonsense.

**Comment by gut1414** [comment]
Pull request problem screenshot. !(https://github.com/user-attachments/assets/d3ba2f4b-ca21-428b-a9cc-e861b0c45205)

**Comment by MagicJinn** [comment]
Currently, you are able to click on the extension icon to submit a new sponsor. The process of new submissions is relatively locked down currently for 3 reasons:
- A rogue agent could add common words like "The" to the pool, and completely decimate every description and comment section
- People who misunderstand this extension could add words that are too general (like Honey and Steam instead of joinhoney and `?utm_source=Influencers`), making it so certain comments and descriptions are removed unnecessarily
- There is currently no infrastructure set up to fetch and push to a database. The strings.json is embedded in the extension itself.
The reason the original SponsorBlock doesn't suffer as much from this issue, is that the segments are submitted on a per-video basis, thus a rogue agent or misinformed user can't ruin the entirety of youtube.

I will keep this issue open, and perhaps rename it to something else, so I don't forget to make it clearer how to report sponsors in comments and descriptions.
